api/v1beta1/database_types.go

```diff
@@ -58,6 +58,7 @@ type DatabaseStatus struct {
 	DatabaseName          string              `json:"database"`
 	UserName              string              `json:"user"`
 	Engine                string              `json:"engine"`
+	Version               string              `json:"version"`
```

Comment:
I would change to `operatorVersion` because it doesn't seem obvious what `version` means